dist/minimongo-observable/to-observable.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var observable_cursor_1 = require('./observable-cursor');
+var _ = require('lodash');
+var COLLECTION_EVENTS_DEBOUNCE_TIMEFRAME = 16;
 function toObservable(cursor) {
     var observable = observable_cursor_1.ObservableCursor.create(function (observer) {
-        var handleChange = function () {
-            observer.next(cursor.fetch());
-        };
+        var rawHandleChange = function () { return Zone.current.run(function () { return observer.next(cursor.fetch()); }); };
+        var handleChange = _.throttle(rawHandleChange, COLLECTION_EVENTS_DEBOUNCE_TIMEFRAME, { trailing: true });
```

Comment:
@dotansimha What do you think if we put _.debounce here instead _.throttle? I feel like it might be a better fit here since  _.throttle always run the callback every 16ms and _.debounce group together based on the 16ms gap.